modules/lazyload/module.php

```diff
@@ -87,6 +87,17 @@ private function apply_dominant_color_background( $control, $value, $selector )
 		return $control;
 	}
 
+	private function is_document_support_lazyload() {
+		$document = \Elementor\Plugin::$instance->documents->get_current();
+		if ( $document ) {
+			$support_lazuyload = $document->get_property( 'support_lazyload' );
+			if ( null !== $support_lazuyload && ! $support_lazuyload ) {
+				return false;
+			}
+		}
+		return true;
```

Comment:
```suggestion
		$document = Plugin::$instance->documents->get_current();
		
		return $document && $document->get_property( 'support_lazyload' );
```

Reply:
This code doesn't work because there is no document yet at that moment. 
the document is null.